Mathlib/Topology/Algebra/Order/LiminfLimsup.lean

```diff
@@ -147,7 +147,43 @@ section LiminfLimsup
 
 section ConditionallyCompleteLinearOrder
 
-variable [ConditionallyCompleteLinearOrder α] [TopologicalSpace α] [OrderTopology α]
+variable [ConditionallyCompleteLinearOrder α]
+
+/-- If `Filter.limsup u atTop ≤ x`, then for all `ε > 0`, eventually we have `u b < x + ε`. -/
+theorem eventually_lt_add_pos_of_limsup_le [Preorder β] [AddMonoid α] [AddLeftStrictMono α]
+    {x ε : α} {u : β → α} (hu_bdd : IsBoundedUnder LE.le atTop u) (hu : Filter.limsup u atTop ≤ x)
```

Comment:
These lemmas don't need topology. Please move to `Order/LiminfLimsup` or (if some imports aren't available there) to a new file in `Order/`.